src/routes/safe/components/Transactions/helpers/TxModalWrapper/index.tsx

```diff
@@ -235,7 +235,7 @@ export const TxModalWrapper = ({
               isExecution={doExecute}
               isRejection={isRejectTx}
               safeNonce={txParameters.safeNonce}
-              txEstimationExecutionStatus={txEstimationExecutionStatus}
+              txEstimationExecutionStatus={safeTxGasError ? EstimationStatus.FAILURE : txEstimationExecutionStatus}
```

Comment:
I assume this is the fix. But why also change the `isCreation` vs `isExecution`?

Reply:
Regarding @francovenica's findings further up. 

I need look at this again though after seeing @usame-algan's comment.